src/Model_Hydro/CPU_Hydro/CPU_Shared_FullStepUpdate.cpp

```diff
@@ -31,6 +31,7 @@
 //
 // Note        :  1. This function is shared by MHM, MHM_RP, and CTU schemes
 //                2. Invoke dual-energy check if DualEnergySwitch is on
+//                3. If any unphysical fluid cells were found in patch, Hydro_FullStepUpdate will return instantly.
```

Comment:
3. If any unphysical fluid cell is found in a patch group, Hydro_FullStepUpdate() will return instantly.

Reply:
Done!